test/factories/mail_notification.rb

```diff
@@ -1,12 +1,21 @@
 FactoryGirl.define do
   factory :mail_notification do
-    sequence(:name) {|n| "notification#{n}"}
-    default_interval "Daily"
-    mailer "HostMailer"
-    mailer_method "test_mail"
-    description "Notifies a user"
-    subscription_type "report"
-    queryable false
+    ignore do
```

Comment:
Since I override the `new` method in `MailNotification`, FactoryGirl needs to be with a custom constructor as it described [here] (https://robots.thoughtbot.com/factory-girl-2-5-gets-custom-constructors)